thenewboston_node/business_logic/models/base.py

```diff
@@ -18,13 +20,30 @@ def get_request_to_block_type_map():
     )
 
     from .signed_change_request import CoinTransferSignedChangeRequest, NodeDeclarationSignedChangeRequest
+
     return {
         CoinTransferSignedChangeRequest: (BlockType.COIN_TRANSFER.value, get_updated_account_states_for_coin_transfer),
         NodeDeclarationSignedChangeRequest:
             (BlockType.NODE_DECLARATION.value, get_updated_account_states_for_node_declaration),
     }
 
 
+NoneType = type(None)
+
+
+def assert_all_optional(cls, *, exempt_fields=None):
```

Comment:
please, reuse BaseMixin code. We need something simple like:

```
assert all(AccountState.is_optional_field(field_name) for field_name in AccountState.get_field_names())
```

Actually that's all we need

Reply:
It looks like this was already implemented in ae91c6e5, so unless you literally just want me to put the assertion in the code, there's nothing more for me to do here.

Reply:
you are about right. but I did not run the code. I need you to make sure that all tests still pass